src/popup/components/TransactionLabel.vue

```diff
@@ -231,7 +230,7 @@ export default defineComponent({
 
       const translation = transactionListTypes[txType.value!] || transactionTypes[txType.value!];
 
-      if (txType.value?.includes('name')) {
+      if (txType?.value?.includes('name')) {
```

Comment:
https://github.com/aeternity/superhero-wallet/pull/1856#discussion_r1120054919

Reply:
i reverted to your solution